src/plugins/intel_gna/src/common/gna_target.cpp

```diff
@@ -12,22 +12,29 @@
 
 namespace ov {
 namespace intel_gna {
-namespace common {
+using namespace common;
+namespace target {
 
 static constexpr const char* kGnaTargetUnspecified = "";
 static constexpr const char* kGnaTargetSoftwareEmulation = "GNA_SW_EMULATION";
-static constexpr const char* kGnaTarget1_0 = "GNA_TARGET_1_0";
+static constexpr const char* kGnaTarget1_0 = "GNA_TARGET_1_0_E";
```

Comment:
not clear why for some targets we use GNA_TARGET_1_0 and for another GNA_TARGET_1_0_E
the same for 3_5 and rest...